core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -7,7 +7,7 @@
          uib-tooltip="<[!isAudioAvailableInCurrentLanguage() ? ('I18N_PLAYER_AUDIO_NOT_AVAILABLE_IN' | translate:{languageDescription:getCurrentAudioLanguageDescription()}) : '']>"
          tooltip-append-to-body="true" tooltip-placement="right">
       </i>
-      <div class="slider-section" ng-if="hasPressedPlayButtonAtLeastOnce">
+      <div class="slider-section" ng-if="hasPressedPlayButtonAtLeastOnce && !audioIsAutoGenerated">
```

Comment:
After these changes, the meaning of the variable `hasPressedPlayButtonAtLeastOnce` doesn't remains. Currently, I don't have any idea for this can you please go through it? (maybe you can achieve what you need with a single scope variable (not sure!))

Reply:
ok sir

Reply:
sir can we use this 
```isProgessBarEnable ``` = ``` hasPressedPlayButtonAtLeastOnce``` && ```audioIsAutoGenerated```